docs/reference/how-to/indexing-speed.asciidoc

```diff
@@ -0,0 +1,95 @@
+[[tune-for-indexing-speed]]
+== Tune for indexing speed
+
+=== Use bulk requests
```

Comment:
I'd add a comment here about using several bulk feeders in parallel (and testing the optimal number).  Besides the parallel processing, it should reduce the cost of each fsync.